### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

File: editor/changeset_wrapper.cpp [hunk 1]
```diff
@@ -314,8 +312,7 @@ std::string ChangesetWrapper::TypeCountToString(TypeCount const & typeCount)
         // "library" -> "libraries"
         else if (lastTwo.back() == 'y' && kVowels.find(lastTwo.front()) == std::string::npos)
         {
-          auto const pos = static_cast<size_t>(ss.tellp());
-          ss.seekp(static_cast<typename std::ostringstream::pos_type>(pos - 1));
+          ss.seekp(ss.tellp() - std::ostringstream::pos_type{1});
```
Comment (vng) [inline editor/changeset_wrapper.cpp:315]: Simple ```ss.tellp() - 1``` doesn't work?

File: editor/changeset_wrapper.cpp [hunk 2]
```diff
@@ -329,19 +326,29 @@ std::string ChangesetWrapper::GetDescription() const
...
+  if (!m_error.empty())
+  {
+    if (!result.empty())
+      result.append("; ");
+    result.append(m_error);
   }
   return result;
 }
+
+void ChangesetWrapper::SetErrorDescription(std::string const & error) {
```
Comment (vng) [inline editor/changeset_wrapper.cpp:351]: { from new line

File: editor/osm_editor.cpp [hunk 3]
```diff
@@ -1210,8 +1205,8 @@ FeatureStatus Editor::GetFeatureStatusImpl(FeaturesContainer const & features,
   return featureInfo->m_status;
 }
 
-bool Editor::IsFeatureUploadedImpl(FeaturesContainer const & features, MwmSet::MwmId const & mwmId,
-                                   uint32_t index) const
+bool Editor::IsFeatureUploadedImpl(FeaturesContainer const & features, MwmId const & mwmId,
+                                   uint32_t index)
```
Comment (vng) [inline editor/osm_editor.cpp:1209]: nit: Put in one line of you are already changing it

File: editor/osm_editor.hpp [hunk 4]
```diff
@@ -142,7 +139,7 @@ class Editor final : public MwmSet::Observer
   bool GetEditedFeatureStreet(FeatureID const & fid, std::string & outFeatureStreet) const;
 
   /// @returns sorted features indices with specified status.
-  std::vector<uint32_t> GetFeaturesByStatus(MwmSet::MwmId const & mwmId,
+  std::vector<uint32_t> GetFeaturesByStatus(MwmId const & mwmId,
                                             FeatureStatus status) const;
```
Comment (vng) [inline editor/osm_editor.hpp:143]: nit: One line

File: editor/osm_editor.hpp [hunk 5]
```diff
@@ -217,37 +214,35 @@ class Editor final : public MwmSet::Observer
...
+                                 MwmId const & id) const;
   EditableProperties GetEditablePropertiesForTypes(feature::TypesHolder const & types) const;
 
   bool FillFeatureInfo(FeatureStatus status, editor::XMLFeature const & xml, FeatureID const & fid,
                        FeatureTypeInfo & fti) const;
   /// @returns pointer to m_features[id][index] if exists, nullptr otherwise.
-  FeatureTypeInfo const * GetFeatureTypeInfo(FeaturesContainer const & features,
-                                             MwmSet::MwmId const & mwmId, uint32_t index) const;
+  static FeatureTypeInfo const * GetFeatureTypeInfo(FeaturesContainer const & features,
+                                             MwmId const & mwmId, uint32_t index) ;
```
Comment (vng) [inline editor/osm_editor.hpp:224]: nit: formatting is strange

File: editor/changeset_wrapper.cpp (hunk 1 above)
Comment (biodranik) [reply editor/changeset_wrapper.cpp:315]: No.
image

**Comment by biodranik** [comment]
@vng PTAL

**Review by vng — APPROVED** [review]

**Pull request merged**